### ISSUE TITLE
attachData Function

### ISSUE THREAD

**Issue by jonnyxe** [Bogardo/Mailgun#83]
Would be great to implement the "attachData" function.

**Comment by Bogardo** [comment]
@jonnyxe Are you referring to ['Attaching Data to Messages'](https://documentation.mailgun.com/user_manual.html#attaching-data-to-messages) as mentioned in the API docs?

**Comment by jonnyxe** [comment]
No, I want to attach a PDF file I saved as a string in the database. The standard mail class in Laravel supports a function to attach a file presented as a string - the attachData function. It would be very great if I could use this function with your mailgun class, too! :)

**Comment by Bogardo** [comment]
This package relies on the [mailgun-php](https://github.com/mailgun/mailgun-php) library and unfortunately that library doesn't support this.
You could request this feature there, once it has been added by them it's a minor tweak on my end to add support for it in this package.
